File: FeedbackManagementSystem-Frontend/feedbackApp/src/app/service/login.service.ts
import { Injectable } from '@angular/core';
import { HttpClient } from '@angular/common/http';
import { Login } from '../models/login.model';
import { EmployeeModel } from '../models/employee.model';


@Injectable({
  providedIn: 'root'
})
export class LoginService {

  loginList : Login[] = [];
  

  constructor(public http : HttpClient) {
    
    this.http.get<Login[]>("../assets/login.json")
    .subscribe(data =>{
      this.loginList = data;
    });
   }

   fetchLoginList():Login[]{
     console.log(this.loginList);
    return this.loginList;
   }

   getEmployee(login : Login){
     console.log(login);
     return this.http.post<EmployeeModel>("http://localhost:8035/login/validate",login);
   }
}

File: FeedbackManagementSystem-Backend/LoginMicroservice/src/main/java/com/cg/service/LoginService.java
package com.cg.service;

import com.cg.beans.Employee;
import com.cg.beans.Login;

/**
 * 
 * @author Shivani
 *
 *	Validate Login 
 */
public interface LoginService {
	
	public Employee validate(Login login);

}
